.github/workflows/triage-move-labelled.yml

```
@@ -44,7 +44,13 @@ jobs:
     name: P1 X-Needs-Design to Design project board
     runs-on: ubuntu-latest
     if: >
-      contains(github.event.issue.labels.*.name, 'X-Needs-Design')
+      contains(github.event.issue.labels.*.name, 'X-Needs-Design') &&
+      (contains(github.event.issue.labels.*.name, 'S-Critical') &&
+       (contains(github.event.issue.labels.*.name, 'O-Frequent') ||
+        contains(github.event.issue.labels.*.name, 'O-Occasional')) ||
+       contains(github.event.issue.labels.*.name, 'S-Major') &&
+       contains(github.event.issue.labels.*.name, 'O-Frequent') ||
```

@kittykat Not sure if this is important, just wondering if these two should be grouped:
```suggestion
       (contains(github.event.issue.labels.*.name, 'S-Major') &&
       contains(github.event.issue.labels.*.name, 'O-Frequent')) ||
```

---

Works either way as && takes priority over || but happy to change for clarification